docs/sources/csv.md

```diff
@@ -130,22 +130,22 @@ usa,200,washington
 canada,100,ottawa
 ```
 
-Below screenshot shows the example of csv inline datasource
+The screenshot below shows the example of CSV inline data source:
 
 ![image](https://user-images.githubusercontent.com/153843/92571108-9e0ff800-f27a-11ea-9fe9-9f9dcbd7125a.png#center)
```

Comment:
This is bad alt text but is also probably out of scope for this patch.

https://grafana.com/docs/writers-toolkit/write/image-guidelines/#alt-text